lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -1518,6 +1521,10 @@ class MailboxDashBoardController extends ReloadableController {
   void updateEmailList(List<PresentationEmail> newEmailList) {
     emailsInCurrentMailbox.value = newEmailList;
   }
+  
+  void openSpamMailboxAction(BuildContext context, PresentationMailbox presentationMailbox){
+    dashBoardAction.value = OpenSpamMailboxAction(context, presentationMailbox);
```

Comment:
You can use `dispatchAction` method to update `dashBoardAction`

```suggestion
dispatchAction(OpenSpamMailboxAction(context, presentationMailbox));
```